code/controllers/subsystem/ticker.dm

```diff
@@ -779,3 +781,69 @@ SUBSYSTEM_DEF(ticker)
 		start_at = world.time + newtime
 	else
 		timeLeft = newtime
+
+/datum/controller/subsystem/ticker/proc/load_mode()
+	var/mode = trim(file2text("data/mode.txt"))
+	if(mode)
+		GLOB.master_mode = mode
+	else
+		GLOB.master_mode = "extended"
+	log_game("Saved mode is '[GLOB.master_mode]'")
+
+/datum/controller/subsystem/ticker/proc/save_mode(the_mode)
+	var/F = file("data/mode.txt")
+	fdel(F)
+	F << the_mode
+
+/datum/controller/subsystem/ticker/proc/SetRoundEndSound(the_sound)
+	set waitfor = FALSE
+	round_end_sound_sent = FALSE
+	round_end_sound = fcopy_rsc(the_sound)
+	for(var/thing in GLOB.clients)
+		var/client/C = thing
+		if (!C)
+			continue
+		C.Export("##action=load_rsc", round_end_sound)
```

Comment:
`browse_rsc` is not for sending files to client's RSCs

It's for sending files from the server rsc to a normal directory on the client that IE can access.

`C.Export("##action=load_rsc"` is for sending files to the client's RSC, and atm it's the only way to do that.

Also, I don't think it's a one time thing, it's more that 99% of the time we don't need to use this as the file already exists on the client's RSC. Admin uploaded files are the exception.